src/stitches.config.ts

```diff
@@ -295,4 +295,43 @@ export const {
   },
 });
 
+export const globalStyles = globalCss({
+  'input, textarea': {
+    appearance: 'none',
+    borderWidth: 0,
+    boxSizing: 'border-box',
+    fontFamily: 'inherit',
+    margin: '0',
+    outline: 'none',
+    padding: '0',
+    width: '100%',
```

Comment:
@f1lander ah, this is the issue; i'm going to remove it

Reply:
in retrospect, moving the resets from stitches components to global CSS was an unsafe move and needed more testing before merging, because it also affects all the material UI components

Reply:
You right, I was afraid of moving to global, I like to keep in its own component, I'll do the change, I did not aware that those will change the material ui too

Reply:
already fixed :)